README.md

```diff
@@ -32,148 +32,40 @@ The available options for installing SciPy packages are listed [here](https://sc
 
 See the tutorials for a full working example.
 
-* [Part 1](https://github.com/CDECatapult/fornax/blob/master/notebooks/tutorial/Tutorial%201%20-%20Creating%20a%20Dataset.ipynb) - Download a small graph dataset 
-* [Part 2](https://github.com/CDECatapult/fornax/blob/master/notebooks/tutorial/Tutorial%202%20-%20Making%20a%20Query.ipynb) - Search the dataset using fornax
+* [Part 1](docs/tutorial/tutorial1.ipynb) - Download a small graph dataset
+* [Part 2](docs/tutorial/tutorial2.ipynb) - Search the dataset using fornax
 
-### Tutorial Dependencies
+### Install Tutorial Dependencies (using conda)
 
 The following tutorials use jupyter notebooks to create a worked example.
-We reccomend you use the anaconda python distribution to run the notebooks.
+We recommend you use the anaconda python distribution to run the notebooks.
 
 ```bash
 conda env create -f environment.yml
-pip install -r requirements.txt
 ```
 
-## Database Setup
-
-By default fornax will use an in memory SQlite database.
-
-Alternative databases can be used by setting the environment variable `FORNAX_DB_URL` using the [sqlalchemy database url format](https://docs.sqlalchemy.org/en/latest/core/engines.html).
-SQLite and Postgresql are supported although other databases are untested.
-
-All tables and indicies are initialised at import time if they do not exist already.
-
-## Quick start
-
-```python
-# create a query graph
-with fornax.Connection() as conn:
-
-    query_graph_handle = fornax.GraphHandle.create()
-    query_graph_handle.add_nodes(id_src=[0, 1, 2], label=['Hulk', 'Lady', 'Storm'])
-    query_graph_handle.add_edges([0, 1], [1, 2])
+### Run the Tutorials
 
+```bash
+source activate fornax_tutorial
+cd docs/tutorial
+jupyter-notebook
+```
 
-    # create a target graph
-    target_graph_handle = fornax.GraphHandle.create()
-    target_graph_handle.add_nodes(id_src=comic_book_nodes['id], label=comic_book_nodes['name'])
-    target_graph_handle.add_edges(comic_book_edges['start'], comic_book_edges['end'])
+## Documentation
 
-    matches = [
-        (query_node_id, target_node_id, weight)
-        for query_node_id, target_node_id, weight
-        in string_similarities
-    ]
+### Build the Docs (requires dev dependencies)
 
-    match_starts, match_ends, weights = zip(*matches)
+```bash
+cd docs
+make html

```

Comment:
Needs to install `fornax` before this will work

Reply:
I've updated the README. Commit 4bf4159